docs/scos/dev/front-end-development/202212.0/oryx/oryx-versioning.md

```diff
@@ -72,3 +72,34 @@ Minor releases are fully backward compatible and do not require any developer as
 ### Deprecation practices
 
 If a newer and better alternative to a feature is introduced in a minor release, the old one is deprecated. Deprecated features remain functional until the next major release, in which they are removed completely.
+
+## Feature Flag Versioning
+
+### Overview
+
+Feature Flag Versioning in Oryx enables us to roll out new functionalities such as alterations in page structure, additional component options, and more, while ensuring backward compatibility. The feature versioning is directly aligned with major and minor releases, allowing for a balanced blend of innovation and stability.
+
+### ORYX_FEATURE_VERSION
+
+Utilize the `ORYX_FEATURE_VERSION` environment variable to opt into new functionalities or to lock and maintain compatibility with existing setups.
+
+### Code Optimization
+
+Setting `ORYX_FEATURE_VERSION` optimizes your build by leveraging Dead Code Elimination (DCE). Any code unrelated to the specified version will not be included in the final bundle, making your application leaner.
+
+### Usage
+
+Set the `ORYX_FEATURE_VERSION` environment variable to enable specific versions.
+
+```
+ORYX_FEATURE_VERSION=1.1
+```
+
+By doing so, you can control feature rollouts while optimizing your application's performance.
+
+You can also use `featureVersion` utility in your code or third-party libraries to conditionally enable features.
```

Comment:
Do I understand correctly that `ORYX_FEATURE_VERSION` is only for code, while `featureVersion` is boith for code and third-party libraries?